Gruntfile.js

```diff
@@ -19,6 +19,30 @@ module.exports = function (grunt) {
   var TEST_BASE_DIR = 'test/';
 
   grunt.initConfig({
+    terser: {
+      options: {
```

Comment:
Is there an option to generate a source map?

Reply:
updated source map option